scripts/Diffraction/isis_powder/pearl.py

```diff
@@ -262,7 +371,18 @@ def _get_monitor_spectrum(self, run_number):
             mspectra = 1
         return mspectra
 
-    # Support for old API
+    def _generate_raw_data_cycle_dir(self, run_cycle):
+        if self._skip_appending_cycle_to_raw_dir():
+            return self.raw_data_dir
+        str_run_cycle = str(run_cycle)
+
+        # Append current cycle to raw data directory
+        generated_dir = os.path.join(self.raw_data_dir, str_run_cycle)
+        generated_dir += '/'
```

Comment:
This is support for `pearl_routines` which previously made nasty assumptions such as this. These hacks have been moved into a separate class called `PearlRoutinesWrapper` which can be removed when the aforementioned script is removed. 

Additionally I had to look this up myself apparently Windows accepts these dividers because it silently treats the path as a URI to a file when it encounters them.